.circleci/config.yml

```diff
@@ -164,6 +168,33 @@ workflows:
             - integration_test
             - build_docker
 
+#      - request-preprod-approval:
```

Comment:
Wouldn't usually commit commented out code but following the kotlin template project example. 😄